.github/Pull_Request_Template.md

```diff
@@ -0,0 +1,40 @@
+## Related Issue
+<!--
+Provide information about the issue or bug that is being addressed.
+-->
+
+Closes: #[issue number]
+
+
+## Description of Changes
+<!-- 
+Clearly and concisely describe the modifications made to successfully resolve the assigned issue. Include any pertinent information about new files or any other relevant details.
+
+For example:
+- Added a new function to handle XYZ.
+- Updated the ABC module to fix the bug.
+- Refactored code in the DEF class for improved performance.
+-->
+
+## Checklist:
+
+<!--
+Mark the checkboxes to indicate completion. Example: 
+- [x] My code follows the style guidelines of this project.
+-->
+
+- [ ] My code adheres to the established style guidelines of this project.
+- [ ] I have conducted a self-review of my code.
+- [ ] I have included comments in areas that may be difficult to understand.
+- [ ] I have made corresponding updates to the project documentation.
+- [ ] My changes have not introduced any new warnings.
```

Comment:
- [ ] The code follows the Code Style Guidelines.
- [ ] Docs have been added for the bug fix or new feature
- [ ] Tests have been added for the bug fix or new feature